packages/proxy-fetcher/src/index.ts

```diff
@@ -87,7 +87,7 @@ export const getFetcher = (
       //  In non-debug mode: only on final attempt
       const logFailedRequest = debug ? true : attempt === retryCount + 1
 
-      if (!response.ok) {
+      if (!response.ok && response.status !== 404) {
```

Comment:
Unless I'm reading this wrong, this still retries 404, it just doesn't log them. Is that the intent?

Reply:
If we wanted to abort for 404, on the next line change it to be ` if (logFailedRequest || response.status === 404)`

Reply:
🤦 You're absolutely right. Just updated.